be/src/note/entities/note.entity.ts

```diff
@@ -0,0 +1,28 @@
+import { Entity, Column, PrimaryGeneratedColumn, CreateDateColumn, ManyToOne } from 'typeorm';
+import { ObjectType, Field, Int } from '@nestjs/graphql';
+import { Item } from 'src/item/entities/item.entity';
+
+@ObjectType()
+@Entity('Note')
+export class Note {
+
+    @Field(() => Int)
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @Field()
+    @Column({ nullable: false })
+    content: string;
+
+    @Field()
+    @CreateDateColumn({ type: "timestamp" })
+    createdAt: Date;
+   
+    @Field(() => Int)
+    @Column()
+    itemId: number;
```

Comment:
Please use a relation here instead of doing custom logic
https://typeorm.io/#/many-to-one-one-to-many-relations

Reply:
Jep it can reference item but item cant reference the notes, that's why when getting notes from items is null

Reply:
And note should be unique, returning note should only have one item in it